languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -93029,7 +93030,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                             <exception postag="V (inf|ppa|ppr).*|V.* 3 s" postag_regexp="yes"/></token>
                     </marker>
                 </pattern>
-                <message>Le verbe « \5 » devrait être à la 3ᵉ personne du singulier.</message>
+                <message>Le verbe « \6 » devrait être à la 3ᵉ personne du singulier.</message>
```

Comment:
Fixed wrong token position (again... we should find a solution to check these automatically)

Reply:
yes that would be great :/ I'm not proud to say that I wouldn't knowhow to start